src/Dropdown/dropdown-style.scss

```diff
@@ -295,6 +280,24 @@ $white: #fff;
     }
 }
 
+// Native version
+.custom-native-dropdown {
+    margin-bottom: 8px;
```

Comment:
Is this margin to leave a space between the native dropdown and the error div ?
If yes i suggest to have the margin set on the error div instead, that way the margin will only be added if there is an error, otherwise the dropdown will be displayed normally

Reply:
No this is margin is a default margin for the entire dropdown, error or not, native or regular. the `.custom-dropdown` has the same margin-bottom